dart/packages/fury-core/lib/src/code_gen/analyse/analysis_cache.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:collection';
+import 'package:fury_core/src/code_gen/entity/fields_cache_unit.dart';
+import 'package:fury_core/src/code_gen/meta/impl/type_immutable.dart';
+import 'package:fury_core/src/code_gen/meta/lib_import_pack.dart';
+import 'package:fury_core/src/dev_annotation/donot_modify.dart';
+
+import '../entity/contructor_params.dart';
+
+class AnalysisCache{
+  static final Map<int, ConstructorParams> _clsIdToUnnamedCons = HashMap();
+  static final Map<int, FieldsCacheUnit> _clsIdToFields = HashMap();
+
+  // _typeToTypeSpecUnnullable 和_typeToTypeSpec1Unnullable 这里只是记录分析过的类型，但是nullable信息不在其中，
```

Comment:
Could we use English comments, so that more contributors can collaborate on this?